### PR TITLE
update factories to interop-config 1.0

### PR DESCRIPTION
**Pull request by sandrokeil**
Not sure if it makes sense to add config id support, because of some discussions in the Gitter channel for multiple event stores? 

/cc @codeliner

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     },
     "require-dev": {
         "container-interop/container-interop": "^1.1",
-        "sandrokeil/interop-config": "^0.3",
+        "sandrokeil/interop-config": "^1.0",
```
Comment (prolic) [inline composer.json]: conflict-section missing

**Comment by sandrokeil** [comment]
@prolic Good catch. :+1:

**Comment by codeliner** [comment]
@sandrokeil good point. Config id support would be great

**Comment by sandrokeil** [comment]
@codeliner This would result in a BC break or we implement a check if config id is given. But we should deprecate usage without config id. Or we create a new factory `DoctrineEventStoreAdapterIdFactory.php`. What do you think?

I would perfer to mark usage without config as deprecated with a php error.

**Comment by sandrokeil** [comment]
@codeliner I think we should not add the ConfigId feature here. We can address it for a new major version. It's a special use case and I think we should not "break" others code.

**Comment by sandrokeil** [comment]
This is ready to merge. We add config id support with next major version.

**Pull request merged**